compiler-rt/lib/sanitizer_common/sanitizer_common_interceptors_memintrinsics.inc

```diff
@@ -40,11 +40,17 @@
 #define PLATFORM_HAS_DIFFERENT_MEMCPY_AND_MEMMOVE 1
 #endif  // SANITIZER_APPLE
 
+#ifndef COMMON_INTERCEPTOR_MEMSET_CHECK_IN_APP_MEM
+#define COMMON_INTERCEPTOR_MEMSET_CHECK_IN_APP_MEM(p) (0)
+#endif
```

Comment:
I would slightly prefer we do not expose this into common layer
and just clone COMMON_INTERCEPTOR_MEMSET_IMPL into hwasan and
use

```


if (MemIsApp(UntagAddr(reinterpret_cast<uptr>(p))) && 
   common_flags()->intercept_intrin)                 \
      COMMON_INTERCEPTOR_WRITE_RANGE(ctx, dst, size);     \

```